zetaclient/orchestrator/orchestrator_test.go

```diff
@@ -518,16 +518,15 @@ func createAppContext(t *testing.T, chainsOrParams ...any) *zctx.AppContext {
 	cfg := config.New(false)
 
 	// Mock config
-	cfg.BitcoinConfig = config.BTCConfig{
-		RPCHost: "localhost",
-	}
-
 	for _, c := range supportedChains {
 		if chains.IsEVMChain(c.ChainId, nil) {
 			cfg.EVMChainConfigs[c.ChainId] = config.EVMConfig{Chain: c}
+		} else if chains.IsBitcoinChain(c.ChainId, nil) {
```

Comment:
Can we use switch case?

Reply:
done in commit https://github.com/zeta-chain/node/pull/2870/commits/582d1de93c6f8531141eb15b0a3c952fc6eba8c5